.devcontainer/postCreateCommands.sh

```diff
@@ -1,13 +1,21 @@
 #!/bin/bash
 
-sudo apt-get update 
-sudo apt-get install -y bash-completion vim iputils-ping telnet
+echo "setup php-scoper"
+composer global require humbug/php-scoper
+$(composer config home)/vendor/bin/php-scoper completion bash >> $HOME.bash_completion
+echo "export PATH=$(composer config home)/vendor/bin/:\$PATH" >> $HOME/.bashrc
+export PATH=$(composer config home)/vendor/bin/:$PATH
+
+sudo cp memory.ini /usr/local/etc/php/conf.d/memory.ini
+
+if [[ "$IS_CI" == "true" ]]; then
```

Comment:
the other stuff is just for interactive use, so we don't need that in the ci on github